src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/Licence.kt

```diff
@@ -128,8 +128,16 @@ abstract class Licence(
   @ManyToOne(fetch = FetchType.EAGER)
   @JoinColumn(name = "responsible_com_id", nullable = false)
   var responsibleCom: CommunityOffenderManager? = null,
+
+  @ManyToOne(fetch = FetchType.EAGER)
+  @JoinColumn(name = "updated_by_id", nullable = true)
+  var updatedBy: Staff? = null,
```

Comment:
Do we need to do performance testing on this or is it assumed to be OK as it's the same pattern as we used for `responsibleCom`?

Reply:
Tested this locally and everything set correctly. Had a chat with Louise re testing the migration. Will test in dev but it is preprod, where we can see how performant everything is.